Design-Docs/tools/scs-compliance-check.py

```diff
@@ -180,14 +186,21 @@ def main(argv):
             else:
                 args = dictval(standard, 'check_tool_args')
                 error = run_check_tool(standard["check_tool"], args, verbose, quiet)
+                if output:
+                    standard.update({"result" : error})
             if not optional:
                 errors += error
             if not quiet and "check_tool" in standard:
                 print(f"... returned {error}")
+            if output:
+                report.update({layer : {"versions": [bestversion]}})
             for kwd in standard:
-                if kwd not in ('check_tool', 'check_tool_args', 'url', 'name', 'condition'):
+                if kwd not in ('check_tool', 'check_tool_args', 'url', 'name', 'condition', 'result'):
```

Comment:
result is still not a valid keyword in the standard specification, no?
So we should continue to complain if we find it.

Reply:
You're right! I removed the key and changed how the result is collected. It now complains that `standard.result` is not a valid keyword, but I'm actually only manipulating the (copied) dictionary `report`. I'll have to continue investigating.

Reply:
copy.deepcopy() creates a real copy, but makes the index() function fail.
We don't need it though, it's always 0.